guides/Game Development/Godot.md

```diff
@@ -0,0 +1,15 @@
+## What is a Game Engine?
+
+A game engine is a software development environment designed for people to build video games.Developers use them to create games for consoles, mobile devices, and personal computers.Godot is one such game engine.The process of game development is often economized, in large part, by reusing/adapting the same game engine to create different games or to make it easier to port games to multiple platforms.
+
+In general a game engine provides you a framework where you can focus on making game and you dont have to write the code to do all the underlying things. It has a rendering engine that produces the graphics in 2D or 3D.
+
+## GODOT
+
+Godot is a 2D and 3D cross-platform compatible game engine released as open source software under the MIT license. It was initially developed for several companies in Latin America before its public release.The development environment runs on Windows, macOS, Linux, BSD and Haiku (both 32 and 64-bit) and can create games targeting PC, console, mobile and web platforms and Godot is completely free.
+
+### NODE
+
+Everything in Godot is a node.A node might be anything,it might be a sprite,an animation,a sound.Every node has a name and it has all sorts of properties that define what it is and how it works. It has callback functions which are ways to make a node react to things. You can send commands to the nodes.A node is extendable that is you can make a node do more things.Nodes are arranged in a tree which means that every node is going to have only one parent but any node can have any number of children.Any tree of nodes grouped together in a Godot is called a Scene.Nodes and Scenes are fundamental to how you build everything in Godot. 
+
```

Comment:
Perhaps explain a little bit more on node and more about Godot
Nice article otherwise